.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: CI Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node-version: [8.x, 10.x, 12.x]
```

Comment:
I see this is 10 / 12 / 14 now, should we remove 8 and add 14? https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

Reply:
Good point, I also see the node driver pre-requisite is node 10.x in the documentation. I will update the driver as well (I copied this from there)